src/main/webapp/js/datahandlers/singlepagedatahandler.js

```diff
@@ -66,7 +66,7 @@ class SinglePageDataHandler {
    * @return {*} - The JSON response.
    */
   async fetchJson_(id) {
-    const response = await fetch(this.getRequestPath_(), {'id': id });
+    const response = await fetch(`${this.getRequestPath_()}?id=${id}`);
```

Comment:
Doesn't getRequestPath take an id as an argument?

Reply:
Oh yeah... I added id to a branch basing off from this branch. Just changed it